src/lib/event.js

```diff
@@ -1,16 +1,9 @@
-const uuidv4 = require('uuid/v4');
 const moment = require('moment');
-const { pool } = require('./connectors/db');
-const Repository = require('@envage/hapi-pg-rest-api/src/repository');
+const repo = require('./connectors/repos');
```

Comment:
It feels like this should be a facade to the new service (if possible) rather than connecting to the repo directly

Reply:
done